shared/common-adapters/modal/index.tsx

```diff
@@ -47,6 +47,7 @@ type Props = {
   noScrollView?: boolean // content must push footer to bottom with this on.
   backgroundStyle?: Styles.StylesCrossPlatform
   scrollViewRef?: React.Ref<ScrollView>
+  scrollviewStyle?: Styles.StylesCrossPlatform
```

Comment:
this will go away soon, but still i think we should call it
`scrollViewContainerStyle` so its more accurate